uspv/init.go

```diff
@@ -137,6 +173,15 @@ func (s *SPVCon) Connect(remoteNode string) error {
 		return err
 	}
 	s.WBytes += uint64(n)
+	return nil
+}
+
+// Connect dials out and connects to full nodes.
+func (s *SPVCon) Connect(remoteNode string) error {
+	err := s.GetSeedAdrs(remoteNode)
```

Comment:
Here, GetSeedAdrs() takes remoteNode, but logically it doesn't need a remote node as an argument, because the point of getting the seed addresses is to figure out a remote node.

If GetSeedAdrs() just returns a list of adrs (which sounds like what it would do) then it doesn't need an argument.